base_upflow/models/account_move.py

```diff
@@ -111,7 +127,7 @@ def _prepare_upflow_payment_api_payload(self):
                 "validatedAt": self.date.isoformat(),
                 "customer": {
                     # "id": self.partner_id.commercial_partner_id.upflow_uuid,
-                    "externalId": str(self.partner_id.commercial_partner_id.id),
+                    "externalId": str(self.upflow_commercial_partner_id.id),
```

Comment:
just seen there is one more in `_prepare_upflow_api_payload` that we probably needs to use `upflow_commercial_partner_id` instead of `self.partner_id.commercial_partner_id`

Reply:
Sorry I missed it when I transfer it from old base_upflow